sandbox/sandbox.go

```diff
@@ -135,6 +140,14 @@ func (sb *sandbox) UpdateAccount(addr crypto.Address, acc *account.Account) {
 	s.updated = true
 }
 
+func (sb *sandbox) AnyRecentTransaction(txID tx.ID) bool {
+	if sb.committedTrxs[txID] != nil {
```

Comment:
Can you add test for this? It can include `CommitTransaction` as well in same test.
Scenario, Commit a transaction, and this should return false

Reply:
You may also check:  `AccumulatedFee`